app/policies/issue_policy.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class IssuePolicy < ApplicationPolicy
+  def new?
+    user.present? && user.reporter?
```

Comment:
This is not right place to check existence of user. User authenticate_user before action instead.